docs/troubleshooting/faq.md

```diff
@@ -0,0 +1,119 @@
+---
+id: faq
+title: Frequently asked questions
+---
+
+This is a collection of frequently asked questions. We do our best to ensure
+everything is included in the docs in the relevant places, but sometimes it's
+either hard to explain in the context of the docs or there just isn't a good
```

Comment:
```suggestion
either hard to explain in the context of the docs or there just isn't a good enough
```
?

Reply:
Uh, both the original sentence and the suggestion basically means exactly the same thing. I'll add in enough to make it explicit.

Reply:
Yeah, I just felt it was missing there :sweat_smile: You can fully ignore it.